### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by sspaul1976** [music-assistant/support#1480] [Q&A] [answered]
I have MA showing through a webpage card on a tablet dashboard with a dark theme on the view. MA shows a light theme. Is there a way to force a dark mode?

**Comment by OzGav** [comment, marked as answer]
It just listens to the OS/browser dark mode. So when you set your OS (or browser to dark mode) it will follow that. As fof beta 144 this is selectable in the settings.

**Comment by sspaul1976** [comment]
Thank you.  I'll play around with that and see if I can figure it out.

**Reply by krisnoble** [reply]
Just a heads-up, you might find that the top bar is always white at the moment, regardless of dark/light mode. I've submitted a [PR](https://github.com/music-assistant/frontend/pull/213) to fix that though so hopefully it'll be approved 🤞🏻

**Comment by priit123** [comment]
Same issue here. I have both os and browser set to dark mode, but Music Assistant is still on light mode. Also, it would be nice to have an option to manually choose between light and dark mode.

**Reply by K3rn3l-P** [reply]
I've noticed that Chromium often reproduces this issue on other installations as well, such as Proxmox, etc. Try clearing your cookies (it doesn't work for me) or use Firefox (set in dark mode)... it works for me! I hope I've been helpful.

**Reply by priit123** [reply]
Thanks,
I tested with Firefox, Chrome and Brave and cleared cookies. 
Firefox and Brave are still in light mode.
Chrome shows dark mode.
Also, phone app shows dark mode.

**Reply by K3rn3l-P** [reply]
So,
Does Chrome show dark mode on your phone? If it does...
It seems like the issue is related to your operating system. You should check if you have dark mode enabled on your current computer (I'm not talking about the browser but the operating system theme, Windows/Linux). Sometimes, the operating system theme can also influence the browser configuration, depending on the settings used.
If not, try to cleanly uninstall browsers that don't enable dark mode and then reinstall them.
Personally, I would first try configuring a virtual machine to understand what the problem might be... certainly related to configurations.
Anyway, to keep it short, there are many guides on Google to adapt themes based on the operating system. 
Note.
Example on my browser Chromium on Linux: chrome://flags > #enable-force-dark flag > "Enabled".
I hope I've been helpful.

**Reply by priit123** [reply]
I found the culprit. In Firefox when privacy.resistFingerprinting flag is true then some websites can't read dark/light flag and default to light. Unfortunately, I wouldn't want to turn it off because privacy is more important to me than the dark theme. I would prefer an option in the Music Assistant.

**Reply by OzGav** [reply]
We have added a item to the backlog to think about this in the future. No promises on timeframe....

**Comment by sesardelaisla** [comment]
It would be great to have an option to set/force dark mode manually for Music Assistant only. I'd rather not to have to set dark mode for the whole environment to have dark mode in Music Assistant.

**Comment by OzGav** [comment]
Beta 144 allows for the setting of the dark mode in the settings.

**Comment by sspaul1976** [comment]
This is great. It looks terrific!  Thanks for all the work you guys are
doing on this product!  I love how it is all coming together.  MA is the
best add-on I have for HA.
